### PR TITLE
Fix: ignore dot files in the monitored directory

### LINKED ISSUE #693: jmxtrans should not pickup hidden files
```2018-08-07 16:05:07 [Thread-3] INFO  com.googlecode.jmxtrans.JmxTransformer - Configuration file deleted: /var/lib/jmxtrans/.chef-localhost20180807-2624-skvi6t.json```

### PR DESCRIPTION
**Pull request by afn**
Fixes #693

### CODE REVIEW

**Comment by afn** [comment]
Rebased.

File: jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java [hunk 1]
```diff
@@ -559,26 +565,26 @@ public void run() {
 		}, 1, TimeUnit.SECONDS);
 	}
 
-	@Override
-	public void fileModified(File file) throws Exception {
+	private void handleFileEvent(File file, String event) throws Exception {
 		if (this.isProcessConfigFile(file)) {
-			log.info("Configuration file modified: " + file);
+			log.info("Configuration file " + event + ": " + file);
```
Comment (gquintana) [inline jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java]: Could we you `{}` placeholders instead of concatenation

Comment (afn) [reply jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java]: Fixed.

**Comment by afn** [comment]
Rebased to master. Good to merge?

**Pull request merged**